llama_stack/providers/inline/telemetry/meta_reference/console_span_processor.py

```diff
@@ -16,6 +16,29 @@
 logger = get_logger(name="console_span_processor", category="telemetry")
 
 
+def _mask_sensitive_value(value: str) -> str:
```

Comment:
can we do more structured? we have the `SecretStr` type -- the whole purpose of that type is to prevent it showing up as is. I think we should audit all our api keys types, including the ProviderData business and annotate them as `SecretStr`.